tests/rules/metavar_ellipsis_param.go

```diff
@@ -0,0 +1,32 @@
+
+// ERROR: match
+func foo(int) { }
+
+// ERROR: match
+func foo(x int) { }
+
+// ERROR: match
+func foo(x, y int) { }
+
+// ERROR: match
+func foo(int, string, bool) { }
+
+// ERROR: match
+func bar(int, string, bool) { }
+
+func bar(x int, string, bool) { }
+
+func bar(string, string, bool) { }
+
+func bar(string, int, string, bool) { }
+
+func qux() { }
+
+func qux(int) { }
+
+func qux(x int) { }
```

Comment:
Wait, shouldn't this match? Ellipsis metavariables should match zero or more items.